modules/apps/portal-workflow/portal-workflow-web/src/main/java/com/liferay/portal/workflow/web/internal/display/context/WorkflowDefinitionLinkDisplayContext.java

```diff
@@ -424,6 +424,43 @@ protected Predicate<WorkflowDefinitionLinkSearchEntry> createPredicate(
 		return predicate;
 	}
 
+	protected WorkflowDefinitionLinkSearchEntry
+			createWorkflowDefinitionLinkSearchEntry(
+				WorkflowHandler<?> workflowHandler, Locale locale)
+		throws PortalException {
+
+		return new WorkflowDefinitionLinkSearchEntry(
+			workflowHandler.getClassName(),
+			HtmlUtil.escapeAttribute(workflowHandler.getType(locale)),
+			getWorkflowDefinitionLabel(workflowHandler));
+	}
+
+	protected List<WorkflowDefinitionLinkSearchEntry>
+			createWorkflowDefinitionLinkSearchEntryList()
+		throws PortalException {
```

Comment:
@mxs2  do we need to make this method protected? I mean the one above should be protected to be tested in the unit test but this one is not being tested in the unit test.